thrift/src/main/thrift/atoms/recipe.thrift

```diff
@@ -18,14 +18,14 @@ struct Time {
 
 struct Serves {
   1: required string type
-  2: required i16 from
-  3: required i16 to
+  2: required i16 min
```

Comment:
I'm not sure what the status of recipe atoms is, but if they're still being maintained in atom workshop then any existing data will need to be migrated

Reply:
I would highly recommend against modifying the format, because we have an archive of all the curated recipes which include those fields and if we remove them from the format then you will not be able to parse them anymore. Recipes is one area considered by the business on the medium/long term where we have identified that we need better structured data.

Reply:
I believe only the field is being renamed, it's keeping its index so the data can still be parsed and serialised. However when updating the library we'll need to fix the field name in the code.

I'll be looking at using the thrift fork we've got though, so that might avoid the problem altogether

Reply:
Resolved with the latest version of the plugin, since we now have control over the code generation this isn't an issue anymore.